src/scripts/main.js

```diff
@@ -5,4 +5,47 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
+const table = document.querySelector('.dashboard > tbody');
+
 // write your code here
+for (let i = 0; i < 6; i++) {
+  const row = document.createElement('tr');
+  const rowItemName = document.createElement('th');
+
+  rowItemName.textContent = people[i].name;
+
+  const rowItemGender = document.createElement('th');
```

Comment:
Replace 'th' with 'td' when creating table cells. 'th' elements should be used for headers, not content.